### DISCUSSION TITLE
How to install a newsletter into a blog using RSS feed?

### DISCUSSION THREAD

**Discussion by amahury** [cotes2020/jekyll-theme-chirpy#1948] [Q&A]
### Checklist

- [X] I have read and accepted the [contributing guidelines](https://github.com/cotes2020/jekyll-theme-chirpy/blob/master/docs/CONTRIBUTING.md).

### Description

Hi everyone!

I'm fascinated with the chirpy template. It's so amazing that I have decided to use it as the template for my personal blog. In order to do it I have read a couple of references I found on the internet: 

[1] https://medium.com/@davideiaiunese/the-problem-why-a-newsletter-baae4409a526
[2] https://dev.to/charalambosioannou/create-a-static-webpage-with-a-contact-form-on-github-pages-3532

The problem is that I feel the information explained in such articles is compatible with the current version of chirpy. I have added 

gem "jekyll-feed"

to the Gemfile but I haven't obtained any feed.xml file. I know I could created by myself but I don't have any idea of where to locate this file as such. When I click the RSS symbol in my template I got in return the following screen:

![Screenshot 2024-09-14 140817](https://github.com/user-attachments/assets/8af116be-1698-483a-9df7-e53998516c7b)

I hope someone can help me to learn how to install a newsletter using chirpy. Thank you so much in advance for reading me.

Amahury

**Comment by Gaur4vGaur** [comment]
Hi,
I tried feedburner and it did not work for me?

What exactly are you trying to achieve?
If you are looking to have subscriber on your website you can use follow.it. I have managed to achieve it for mine.

**Reply by sulemanhasib43** [reply]
the code which is provided by follow.it where did you place it on your chirpy installation?

**Comment by Aneeqasif** [comment]
follow.it is your way

**Comment by amahury** [comment]
Thank you so much to both of you! follow.it worked exactly for what I
wanted.

**Reply by Aneeqasif** [reply]
where's your blog living

**Comment by amahury** [comment]
https://amahury.github.io/

Currently a partial solution I'm using is Brevo. However, I'm planning to
use follow.it to improve my blog.
